insertion-sort.md

```diff
@@ -0,0 +1,100 @@
+# Insertion Sort
+
+Insertion sort is a comparision based sorting. In this sorting technique, we maintain a sub-list which is always sorted and then we take one element from the list and insert it at its correct place. We does so till all elements are inserted into sublist. For example, while playing cards we sort cards in our hand. Starting from left and moving to right,we keep on inserting the card at it's right place till end.
```

Comment:
> Insertion sort is a comparision based sorting

Add a line here about what is a comparison based sorting. You can link to some external Wiki article too.